deb-get

```diff
@@ -707,6 +707,14 @@ function deb_jellyfin() {
     WEBSITE="https://jellyfin.org/"
 }
 
+function deb_pandoc() {
+    ARCHS_SUPPORTED="amd64 arm64"
+    get_github_releases "https://api.github.com/repos/jgm/pandoc/releases"
+    VERSION_PUBLISHED="$(grep "browser_download_url.*${HOST_ARCH}.deb" "${CACHE_DIR}/${APP}.json" | head -n1 | head -n1 | cut -d'"' -f4 | cut -d'-' -f2)"
```

Comment:
Just happened to spot a superfluous `head -n1` in `head -n1 | head -n1`.

Reply:
Thanks. That's what happens with copy-pasta late-night contributing. Already merged though, so maybe a PR to trim down a few entries would be in order (it's also in ` deb_balena-etcher-electron`and `deb_figma-linux`).